script.js

```diff
@@ -0,0 +1,35 @@
+// Requirements:
+// 1. user should click on different elements to create a move
+// 2. every click alternates between x and o
+// 3. when a cell is marked, display a corresponding color
+// 4. marked cell can't be undone
+// 5. add reset button will clear contents
+
+// Outline:
+// 1. listen for a click event on a box
+// 2. when click on a box is executed, toggle between marking x and o
+
+
+var playerMoves = [];
+var storePlayerMoves = playerMoves.push();
```

Comment:
not really sure what this line is doing. `.push()` should have an argument that you want added to an array. The use case of `storePlayerMoves` is essentially already encapsulated by `playerMoves`